ingestion/src/metadata/ingestion/source/database/dbt/metadata.py

```diff
@@ -721,7 +721,7 @@ def process_dbt_meta(self, manifest_meta):
                     get_tag_labels(
                         metadata=self.metadata,
                         tags=dbt_meta_info.openmetadata.glossary,
-                        include_tags=self.source_config.includeTags,
+                        include_tags=True,
```

Comment:
I don't think this should be an expected behaviour, what if user does not want to include both tags and glossaries, we should not force this ingestion

cc: @OnkarVO7

Reply:
For Tags we have the `includeTag` rule.
For glossaries, users have to make specific changes on their `dbt yml` files.

Reply:
To be able to ingest Glossaries these two must be satisfied
- Glossary must be created beforehand on Openmetadata.
- These Glossary FQN must be passed on the `meta` section on the `schema.yml` file on `dbt`.